ow_faults/src/FaultInjector.cpp

```diff
@@ -12,9 +12,13 @@ constexpr std::bitset<10> FaultInjector::isPanTiltExecutionError;
 constexpr std::bitset<10> FaultInjector::isArmExecutionError;
 constexpr std::bitset<10> FaultInjector::isPowerSystemFault;
 
+constexpr std::bitset<3> FaultInjector::islowVoltageError;
+constexpr std::bitset<3> FaultInjector::isCapLossError;
+constexpr std::bitset<3> FaultInjector::isThermalError;
 
 FaultInjector::FaultInjector(ros::NodeHandle node_handle)
 {
+  //  arm pub and subs
   auto joint_states_str = "joint_states";
```

Comment:
Minor comment: making this a static (file) variable, and then using a class initializer list instead of all the following assignments, is cleaner and a bit more efficient.

Reply:
The variable is not used beyond this function .. so I wouldn't recommend converting it to a static (file) variable

Reply:
also I think that the auto type is deduced to const char * which means this is a const value rather than a variable.

Reply:
Good point that 'const char*' is a better type.  The reason I suggested file variable is so that a class initializer list could be used in the constructor.  That's evaluated before the function body.